### PR TITLE
Make an spi out of the keyStringResolver

### PR DESCRIPTION
**Pull request by acdemyers**

### CODE REVIEW

**Comment by acdemyers** [comment]
#build

**Comment by acdemyers** [comment]
#build

**Comment by acdemyers** [comment]
#build

**Comment by acdemyers** [comment]
#build

**Comment by acdemyers** [comment]
#build

**Comment by acdemyers** [comment]
#build

**Comment by acdemyers** [comment]
#build

**Comment by acdemyers** [comment]
#build

**Comment by acdemyers** [comment]
#build

**Comment by acdemyers** [comment]
#build

**Comment by acdemyers** [comment]
#build

File: dev/com.ibm.ws.crypto.passwordutil/src/com/ibm/wsspi/security/crypto/package-info.java [hunk 1]
```diff
@@ -11,5 +11,5 @@
 /**
  * @version 1.1.0
```
Comment (awisniew90) [inline dev/com.ibm.ws.crypto.passwordutil/src/com/ibm/wsspi/security/crypto/package-info.java]: We should up this to 1.2.0 as well

**Review by awisniew90 — APPROVED** [review]
One comment to change, but looks good!

**Comment by acdemyers** [comment]
#build

**Comment by acdemyers** [comment]
#run-libby-bot

**Pull request merged**